azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -79,6 +82,14 @@ public static ExecutableFlow createExecutableFlowFromObject(final Object obj) {
     return exFlow;
   }
 
+  public AzkabanFlow getAzkabanFlow() {
+    return azkabanFlow;
+  }
+
+  public void setAzkabanFlow(AzkabanFlow azkabanFlow) {
```

Comment:
should AzkabanFlow be initialized from constructor?

Reply:
I guess this might be necessary for migration purpose.

Reply:
Yes, this AzkabanFlow will only apply to flow.2.0, and we still need to keep backward compatibility.